### PR TITLE
feat: add outfits validations

### PR DESCRIPTION
**Pull request by pedrotambo**

### CODE REVIEW

File: src/validations/access/common/profile.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import { ADR_74_TIMESTAMP, ADR_75_TIMESTAMP } from '../../timestamps'
+import { validateAfterADR75 } from '../../validations'
+
+export function createNamesOwnershipValidateFn(
+  { externalCalls }: Pick<ContentValidatorComponents, 'externalCalls'>,
+  namesOwnership: NamesOwnership
+): ValidateFn {
+  async function validateFn(deployment: DeploymentToValidate) {
+    const ethAddress = externalCalls.ownerAddress(deployment.auditInfo)
+    const names = deployment.entity.metadata.avatars
```
Comment (marianogoldman) [inline src/validations/access/common/profile.ts:23]: I think names have to be lowercase when querying the-graph

File: src/validations/access/on-chain/profiles.ts [hunk 2]
```diff
@@ -1,113 +1,10 @@
...
+import { OnChainAccessCheckerComponents } from '../../../types'
+import { validateAll } from '../../validations'
+import { createItemOwnershipValidateFn, createNamesOwnershipValidateFn } from '../common/profile'
 
 export function createProfileValidateFn(components: Pick<OnChainAccessCheckerComponents, 'client' | 'externalCalls'>) {
   return validateAll(
-    createNamesOwnershipValidateFn(components),
-    createItemOwnershipValidateFn(components),
-    createPointerValidateFn(components)
+    createNamesOwnershipValidateFn(components, components.client),
```
Comment (marianogoldman) [inline src/validations/access/on-chain/profiles.ts:7]: Shouldn't the receiver function get the client out of components?

File: src/validations/access/common/profile.ts (hunk 1 above)
Comment (marianogoldman) [reply src/validations/access/common/profile.ts:23]: Sorry, no... not the case. The query needs to be done in the exact same case as the name was minted.

Comment (pedrotambo) [reply src/validations/access/common/profile.ts:23]: On the contrary names in the graph are case sensitive :(. In addition, this is just moved code, no logic added

File: src/validations/access/on-chain/profiles.ts (hunk 2 above)
Comment (pedrotambo) [reply src/validations/access/on-chain/profiles.ts:7]: Yeah, but as this is a "common" validate fn, the component depends if the validator is TheGraph or OnChain

**Pull request merged**